src/machine/image-dbus.c

```diff
@@ -17,6 +17,7 @@
   along with systemd; If not, see <http://www.gnu.org/licenses/>.
 ***/
 
+#include <sys/file.h>
```

Comment:
This include is for `LOCK_*` macros. Though I'm not sure where should they normally be defined. glibc defines it in `fcntl.h` if `_DEFAULT_SOURCE` or `_GNU_SOURCE` is defined.